src/app/book/book.js

```diff
@@ -1,14 +1,48 @@
-(function () {
+(function() {
   'use strict';
 
-  function BookCtrl ($location) {
+  function BookCtrl() {
+    var date       = $('#date');
+    var startTime  = $('#time-start');
```

Comment:
you shouldn't mix jquery with angular.  angular already comes with jquery lite
jquery lite is `angular.element('#time-start')`